### PR TITLE
General cleanup

### PR DESCRIPTION
**Pull request by brian-avery**
General cleanup. 
* separated the Maistra and operator version
* Converted readme.adoc to asciidoc so that it can use our tooling as well. 
* Added Kiali to installation options updater so that we can auto-update our Kiali install options as well

### CODE REVIEW

File: themes/hugo-theme-learn-master/layouts/partials/menu.html [hunk 1]
```diff
@@ -5,6 +5,9 @@
   <div id="header-wrapper">
     <div id="header">
       {{ partial "logo.html" . }}
+      <font style="color:#fff">
```
Comment (jwendell) [inline themes/hugo-theme-learn-master/layouts/partials/menu.html]: Could you increase the font, or make it bolder? I barely can see it :)

File: tools/updateFields.sh [hunk 2]
```diff
@@ -1,41 +1,75 @@
 #!/bin/sh
-set -x
+#set -x
```
Comment (jwendell) [inline tools/updateFields.sh]: Remove this line?

Comment (jwendell) [reply tools/updateFields.sh]: Also, setting `-e` is a good practice (can be done later, of course)

File: tools/updateFields.sh [hunk 3]
```diff
@@ -1,41 +1,75 @@
...
+    yq merge out/base out/maistra > out/maistra.rendered
+    yq merge out/base out/servicemesh > out/servicemesh.rendered
+}
+
+getKiali() {
+    file=out/kiali.rendered
+
+    wget https://raw.githubusercontent.com/kiali/kiali-operator/master/deploy/kiali/kiali_cr.yaml -O ${file}
+
+    MatchingLines=$(grep -nr "#[[:blank:]]* ---" ${file} | cut -d : -f1 )
```
Comment (jwendell) [inline tools/updateFields.sh]: My opinion here: I don't like this style. In my scripts I tend to follow the conventions of lower case for local variables, UPPER CASE for globals. No need to worry about that now, you can always improve it in a follow up. Hint: use `local` for local scoped variables.

File: README.adoc [hunk 4]
```diff
@@ -3,15 +3,28 @@
...
+Maistra has a variety of tooling located in the tools directory. 
 
-The OpenShift Service mesh documentation is written in [AsciiDoc](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/). This structure of this repository has been designed to work with both the upstream and downstream toolsets. To generate the documentation, you should install the following tools:
+* CheckLinks.sh
+CheckLinks is a wrapper around the html-proofer tool that runs on pull requests to make sure that links are valid. 
 
-* [Hugo](https://gohugo.io/)
-* [Asciidoctor](http://asciidoctor.org/docs/install-toolchain/). The downstream (product) documentation toolset requires AsciiDoc, and while the main content format for Hugo is markdown, Hugo also [supports AsciiDoc files](https://gohugo.io/content-management/formats/#additional-formats-through-external-helpers) by calling Asciidoctor.
-* Text editor of your choice, for example the [Atom](https://atom.io/) text editor has several useful [packages](https://atom.io/packages) that make it easier to work with AsciiDoc and Hugo.
+* RunLinter.sh
```
Comment (jwendell) [inline README.adoc]: I think it is `runLinter.sh`?

File: README.adoc [hunk 5]
```diff
@@ -3,15 +3,28 @@
 
 This repository contains documentation for the community release of OpenShift Service Mesh, based on the upstream Istio project. This README provides information about this documentation repository. The site can be viewed at maistra.io.
 
-## Getting Started
+== Tooling
+Maistra has a variety of tooling located in the tools directory. 
 
-The OpenShift Service mesh documentation is written in [AsciiDoc](http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/). This structure of this repository has been designed to work with both the upstream and downstream toolsets. To generate the documentation, you should install the following tools:
+* CheckLinks.sh
```
Comment (jwendell) [inline README.adoc]: `checkLinks.sh`?

**Review by jwendell — APPROVED** [review]
LGTM, made a few nit comments that you can improve in a follow up.

I'd love to see something like this in `Makefile` too:

```Makefile
lint:
	find . -name '*.sh' -print0 | xargs -0 -r shellcheck
```

File: README.adoc (hunk 5 above)
Comment (brian-avery) [reply README.adoc]: Fixed. Thanks.

File: README.adoc (hunk 4 above)
Comment (brian-avery) [reply README.adoc]: Fixed. Thanks.

File: tools/updateFields.sh (hunk 2 above)
Comment (brian-avery) [reply tools/updateFields.sh]: Fixed. Thanks.

File: themes/hugo-theme-learn-master/layouts/partials/menu.html (hunk 1 above)
Comment (brian-avery) [reply themes/hugo-theme-learn-master/layouts/partials/menu.html]: How's it look now?

File: tools/updateFields.sh (hunk 3 above)
Comment (brian-avery) [reply tools/updateFields.sh]: ^--------^ SC2039: In POSIX sh, 'local' is undefined.

Comment (jwendell) [reply tools/updateFields.sh]: use bash, not sh. You can also run shellcheck locally before pushing to save you some time.

File: themes/hugo-theme-learn-master/layouts/partials/menu.html (hunk 1 above)
Comment (jwendell) [reply themes/hugo-theme-learn-master/layouts/partials/menu.html]: It looks way better :)

Comment (jwendell) [reply themes/hugo-theme-learn-master/layouts/partials/menu.html]: Can we have a link/menu/dropdown/whatever for other versions like Istio does?

Comment (jwendell) [reply themes/hugo-theme-learn-master/layouts/partials/menu.html]: not for this pr

Comment (brian-avery) [reply themes/hugo-theme-learn-master/layouts/partials/menu.html]: That's an idea I'm playing with in my head. That information is present in both the release info page and the drop down. may make sense to have that information stored in an object and auto-generate the table... not sure yet.

**Comment by brian-avery** [comment]
Since we're adding shellcheck anyway, I wonder if it's worth it to extract any bash from the asciidoc and verify that as well.

File: Makefile [hunk 6]
```diff
@@ -16,3 +16,9 @@ lint:
 	./tools/runLinter.sh
 check-links:
 	./tools/checkLinks.sh
+gen-install-options:
+	./tools/updateFields.sh vars.adoc
+serve: gen-install-options
+	hugo serve
+build: gen-install-options lint check-links
+	hugo -D
```
Comment (dgn) [inline Makefile:24]: nit: let's mark all these as `.PHONY` so we don't run into strange issues when somebody creates a file or folder with the same name

**Review by dgn — APPROVED** [review]

**Comment by brian-avery** [comment]
Done. Thanks.

**Pull request merged**